core/src/plugins/kubernetes/helm/status.ts

```diff
@@ -205,19 +205,25 @@ export async function getReleaseStatus({
 
     if (state === "ready") {
       // Make sure the right version is deployed
-      values = JSON.parse(
-        await helm({
-          ctx,
-          log,
-          namespace,
-          args: ["get", "values", releaseName, "--output", "json"],
-          // do not send JSON output to Garden Cloud or CLI verbose log
-          emitLogEvents: false,
-        })
-      )
-
-      const deployedVersion = values[".garden"]?.version
-      deployedMode = values[".garden"]?.mode
+      const helmResponse = await helm({
+        ctx,
+        log,
+        namespace,
+        args: ["get", "values", releaseName, "--output", "json"],
+        // do not send JSON output to Garden Cloud or CLI verbose log
+        emitLogEvents: false,
+      })
+      values = JSON.parse(helmResponse)
+
+      let deployedVersion: string | undefined = undefined
+      // JSON.parse can return null
+      if (values !== null) {
```

Comment:
```suggestion
      if (values === null) {
```
Don't we want to check if the values are null and return `state: "outdated"` if that's the case? Trying to get the `deployedVersion` from the values will still fail.

Reply:
I thought about that too. Now, if the `values` is `null`, it will be handled in [L226](https://github.com/garden-io/garden/pull/5307/files#diff-9e897b08cb49cc822f3496aa64e103cad793f9414b6f34960bd21416acf66785R226).

Of course, we can explicitly set the status to `outdated` if the `values` is `null`. Just for better clarity.

Reply:
Changed this in 6ac81b4396f73d6fb8a3c9c82b3576585f658e64.